treeherder/model/exchanges.py

```diff
@@ -32,6 +32,26 @@ class TreeherderPublisher(PulsePublisher):
         schema="https://treeherder.mozilla.org/schemas/v1/resultset-message.json#"
     )
 
+    resultset_action = Exchange(
+        exchange="resultset-actions",
+        title="Actions issued by resultset",
+        description="""
+            There are actions which can be done to a resultset
+            (fill-in) they are published on this exchange
+        """,
+        routing_keys=[
+            Key(
+                name='project',
+                summary="Project (or branch) that this result-set concerns"
```

Comment:
I would word it more like its job_action counterpart:
Project which this result set belongs to